content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,285 @@
+Manage lots for shipping and receiving
+======================================
 
-.. image:: lots/lots_06.png
-    :align: center
+Lot numbers can be assigned for both **incoming** and **outgoing** goods. For incoming goods, lot
+numbers are assigned directly on the purchase order form. For outgoing goods, lot numbers are
+assigned directly on the sales order form.
 
-.. image:: lots/lots_05.png
-    :align: center
+Manage lots on receipts
+-----------------------
 
-.. image:: lots/lots_07.png
-    :align: center
+Assigning lot numbers to **incoming** goods can be done directly from the purchase order (PO).
 
-Operation Types
----------------
+To create a :abbr:`PO (purchase order)`, go to the :menuselection:`Purchase app --> Create`. Doing
+so reveals a new, blank request for quotation (RFQ) form.
 
-Of course, you also have the possibility to define how you will manage
-lots for each operation type. To do so, open the *Inventory* app and
-go to :menuselection:`Configuration --> Operation Types`.
+On this :abbr:`RFQ (request for quotation)`, fill out the necessary information by adding a
+:guilabel:`Vendor`, and adding the desired products to the :guilabel:`Product` lines, by clicking
+:guilabel:`Add a product` (under the :guilabel:`Products` tab).
 
-For each type (receipts, internal transfers, deliveries, …), you can
-decide if you allow to create new lot numbers or only use existing ones.
-By default, the creation of new lots is only allowed at product
-reception, and using existing lot numbers isn’t allowed in this
-operation. If you have inter-warehouse transfers and track products by
-lots, it can be useful to allow using existing lot numbers in receipts
-too.
+Choose the desired quantity of the product to order by changing the number in the
+:guilabel:`Quantity` column.
 
-.. image:: lots/lots_08.png
-    :align: center
+Once the :abbr:`RFQ (request for quotation)` has been filled out, click :guilabel:`Confirm Order`.
+When the :abbr:`RFQ (request for quotation)` is confirmed, it becomes a :guilabel:`Purchase Order`,
+and a :guilabel:`Receipt` smart button appears. Click the :guilabel:`Receipt` smart button to be
+taken to the warehouse receipt form.
+
+.. note::
+   Clicking :guilabel:`Validate` before assigning a lot number to the ordered product quantities
+   will result in a :guilabel:`User Error` pop-up. The pop-up requires entry of a lot or serial
+   number for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be validated
+   without a lot number being assigned.
+
+.. image:: lots/lots-user-error-popup.png
+   :align: center
+   :alt: Add lot/serial number user error popup.
+
+From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
+horizontal lines, located to the right of the :guilabel:`Unit of Measure` column in the
+:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```

Comment:
The active element, the icon, should be in a guilabel and I believe it would be appropriate to put the icon descriptor in the parenthesis
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`hamburger
(four horizontal lines)` icon, located to the right of the :guilabel:`Unit of Measure` column in the
:guilabel:`Operations` tab. Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```